utilities.py

```diff
@@ -8,30 +14,41 @@
 
 
 def parameter_file_load(x):
-    try:
-        with open(x) as json_file:
-            text_input = pd.read_json(json_file)
-            return text_input
-    except IOError:
-        print("File Handling Error: ", IOError)
+    if os.path.exists(x):
+        try:
+            with open(x) as json_file:
+                text_input = pd.read_json(json_file)
+                return text_input
+        except Exception as e:
+            logging.error("There was issue to Open\Read the file Due to: " + str(e))
+    else:
+        logging.error("Could not find the query file in " + os.path.dirname(x))

```

Comment:
again you need to raise exception here in order to fail

Reply:
statement removed due to changes in the conversation above.